meson.build

```
@@ -69,7 +69,7 @@ subdir('src/lua')
 
 inc = [ include_directories('include', 'include/taskolib/sol', 'src') ]
 
-gul_dep = dependency('libgul14', version : '> 2.6', fallback : ['libgul14', 'libgul_dep'])
+gul_dep = dependency('gul17', version : '> 2.6', fallback : ['gul17', 'libgul_dep'])
```

I believe with GUL17 we have `yy.mm.nn` versions.
And the first GUL17 was "GUL14 v2.12.1+", so this can go. 

```suggestion
gul_dep = dependency('gul17', fallback : ['gul17', 'libgul_dep'])
```